### PR TITLE
added italian translation

### PR DESCRIPTION
**Pull request by fravaccaro**

### CODE REVIEW

File: translations/harbour-phillis-it.ts [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+        <location filename="../qml/pages/SettingsPage.qml" line="230"/>
+        <source>Require a PIN to access the app</source>
+        <translation>Richiedi un PIN per accedere all&apos;app</translation>
+    </message>
+    <message id="ph-settings-page-access-restrict-description">
+        <location filename="../qml/pages/SettingsPage.qml" line="232"/>
+        <source>Protect against accidently accessing the application. The PIN will be stored as plain text in the application configuration directory. Should you forget your PIN simply delete the file to restore access to the application.</source>
+        <translation>Proteggi contro l&apos;accesso accidentale all&apos;app. Il PIN sarà salvato come file di testo nella cartella di configurazione dell&apos;app, per cui nel caso dovessi dimenticarlo ti basterà cancellare quel file per ripristinare l&apos;accesso all&apos;app.</translation>
+    </message>
+    <message id="ph-settings-page-access-require-pin-label">
```
Comment (grumpycat3051) [inline translations/harbour-phillis-it.ts:111]: What about the rest of the strings?

**Review by grumpycat3051 — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please translate the rest of the strings.

File: translations/harbour-phillis-it.ts (hunk 1 above)
Comment (fravaccaro) [reply translations/harbour-phillis-it.ts:111]: what do you mean?

**Comment by fravaccaro** [comment]
Done :) I'm sorry if I missed some in the first place

**Comment by grumpycat3051** [comment]
Thank you for your help. I have merged your changes. There where some changes to the about page and the newly added translations page that you might want to look at.

**Pull request closed without merging**